ocaml/doc/wire-protocol.md

```diff
@@ -639,7 +656,7 @@ To retrieve the record of a single object as a python dictionary:
 To retrieve all the VM records in a single call:
 
 ```python
->>> records = client.call("VM.get_all_records", session)
+>>> records = jsonrpccall("VM.get_all_records", (session,))
 >>> records.keys()
```

Comment:
Arriving a bit too  late to this, and not caused by this PR, but the output shown is retrieved with `list(records.keys())`.
`records.keys()` shows `dict_keys(['OpaqueRef:1', 'OpaqueRef:2', 'OpaqueRef:3', 'OpaqueRef:4' ])`